tsconfig.compileroptions.json

```diff
@@ -11,6 +11,7 @@
     "composite": true,
     "declaration": true,
     "sourceMap": true,
+    "skipLibCheck": true,
```

Comment:
this may be controversial with the former ember-TS members, but I don't know of another way to have a node16 project interoperable with non-none16 projects

Reply:
I would prefer to fix this issue upstream in the glimmer side. `skipLibCheck` is a pretty blunt tool that can hide problems globally (since type failures in a library can generate `any`s that then propagate through our own code).

Reply:
https://github.com/glimmerjs/glimmer-vm/pull/1555